core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.spec.ts

```diff
@@ -131,6 +135,30 @@ describe('Contributor dashboard page', function() {
       expect(ctrl.activeTabName).toBe(changedTab);
       expect(ctrl.showLanguageSelector()).toBe(true);
     });
+
+    it('should call scrollFunction on scroll', function() {
+      var e = document.createEvent('Event');
+      var scrollSpy = spyOn(ctrl, 'scrollFunction');
+      e.initEvent('scroll', true, true);
+      window.dispatchEvent(e);
+      expect(scrollSpy).toHaveBeenCalled();
+    });
+
+    it('should not be viewed the collapsed header content if' +
+      ' scrolled less than 5', function() {
+      mockWindow.scrollTop = 3;
+      ctrl.scrollFunction();
+      expect(ctrl.defaultHeaderVisible).toBe(true);
+      expect(ctrl.collapsibleHeaderVisible).toBe(false);
+    });
+
+    it('should not be viewed the collapsed header content if' +
+      ' scrolled less than 5', function() {
```

Comment:
This test name is the same as the above? In general, please make sure to do a quick sanity check on your changes.

Reply:
Done